dist/definitions.json

```diff
@@ -335,5 +335,133 @@
     "enum": [
       true
     ]
+  },
+  "gender": {
+    "type": "string",
+    "enum": [
+      "M",
```

Comment:
Can you swap the order of these? We've been ordering this Female, Male on the front end, so I'll end up changing it when we build the form anyway.